docs/examples/pipeline/query_pipeline_sql.ipynb

```diff
@@ -570,8 +570,35 @@
     "from pyvis.network import Network\n",
```

Comment:
<p>very nit: "agent_dag.html"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/11880/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
yeah sorry forgot to change that name